src/update.c

```diff
@@ -622,6 +625,7 @@ static void print_help(const char *name)
 	fprintf(stderr, "   -n, --nosigcheck        Do not attempt to enforce certificate or signature checking\n");
 	fprintf(stderr, "   -I, --ignore-time       Ignore system/certificate time when validating signature\n");
 	fprintf(stderr, "   -S, --statedir          Specify alternate swupd state directory\n");
+	fprintf(stderr, "   -k, --keepcache         Do not delete the swupd state directory after updating the system\n");
```

Comment:
We don't really delete the state directory. Can you say "delete the swupd state directory content ..." or "Do not clean the swupd state directory cache..."

Reply:
sure